src/components/ProfileEditPOWnamePage.js

```diff
@@ -43,11 +43,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
+//async somewhere here
```

Comment:
Yes, you should await somewhere here.

But first, you should create an error state. So instead of `const error = 1` you should have const [error, setError] = useState(false). 

Then you should await `updateUser` inside `createNewPOWname`. If the result is an object with an error. Ie. `if(result.error)` then you should `setError(result.error)`, and if not `setError(false)`.

Later you should use the `error` state to set the `helperText`prop on the TextField to `error.message` and set the `error` to `true` so the styling turns error red.

Give it another go, and if you are still having problems I will make a video for you to follow.

Reply:
I should await somewhere here.

But first, you should 
create an error state. So instead of `const error = 1` you should have 

- [ ] const [error, setError] = useState(false). 

- [ ] Then you should 
- [ ] await `updateUser` inside `createNewPOWname`. 
- [ ] If the result is an object with an error. Ie. 
- [ ] `if(result.error)` then you should 
- [ ] `setError(result.error)`, and 
- [ ] if not `setError(false)`.
- [ ] 
- [ ] Later you should 
- [ ] use the `error` state to 
- [ ] set the `helperText`prop on 
- [ ] the TextField to `error.message` and 
- [ ] set the `error` to `true` so the styling turns error red.

Give it another go, and if you are still having problems I will make a video for you to follow.

Reply:
I tried to  "undo last commit" on the four commits above, the commits I am writing about above, but just when I thought I was out, They Pull Me Back In.


 (may 5)

f2bba4d
 6c054ae
69566ab
dc12c87